.env.template

```diff
@@ -1,21 +1,3 @@
-API_URL=
-API_TOKEN=
-USE_MOCK_SERVICES=true
-
-# splunk service variables
-SPLUNK_HEC_TOKEN=
-SPLUNK_PASSWORD=
-SPLUNK_COLLECTOR_URL=
-
-# github credentials
-GITHUB_URL=
-GITHUB_TOKEN=
-GITHUB_REPOSITORY=
-GITHUB_TOKEN=
-
-#redis credentials
-REDIS__PASSWORD=
-REDIS__HOST=
-
+# Azure Form Recognizer
 FORMRECOGNIZER__APIKEY=00000000000000000000000000000000
-FORMRECOGNIZER__ENDPOINT=https://westus2.api.cognitive.microsoft.com/
+FORMRECOGNIZER__ENDPOINT=https://someserver.api.cognitive.microsoft.com/
```

Comment:
I dont think the server name here needs to be removed. The zones and regions are well known and documemented. https://docs.microsoft.com/en-us/azure/availability-zones/az-overview

Reply:
Okay, then I'll leave it.